src/main/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisation.java

```diff
@@ -23,14 +24,21 @@
 
     public Map<String, String> setRepresentativeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
         if (isValidReps(sscsCaseData.getAppeal().getRep())) {
-            personalisation.put(AppConstants.REPRESENTATIVE_NAME, String.format("%s %s",
-                    sscsCaseData.getAppeal().getRep().getName().getFirstName(),
-                    sscsCaseData.getAppeal().getRep().getName().getLastName()));
+            personalisation.put(AppConstants.REPRESENTATIVE_NAME, 
+                    getDefaultName(sscsCaseData.getAppeal().getRep().getName(),
+                    "Sir / Madam"));
```

Comment:
See if you can use the existing uk.gov.hmcts.reform.sscs.service.SendNotificationService#getRepSalutation method.

Reply:
I did take a look and that service does a lot of stuff with notificationWrappers which aren't available down here at the Personalisation level :-(

Reply:
Refactored the Service slightly so that I can call it without a wrapper and made the above code call it to re-use the existing code.